pyaedt/modeler/cad/Primitives3D.py

```diff
@@ -3339,3 +3339,96 @@ def _check_value_type(self, taken_value, value_type, are_inequations_checkable,
                 )
                 are_inequations_checkable = False
         return receiving_variable, are_inequations_checkable
+
+    @pyaedt_function_handler()
+    def create_geometry_from_csv(self, csv_file, header_line=2, column_mapping=None, geo_type="cylinder", unit="mm"):
+        """Create geometry from csv file containing geometry data.
+        Parameters
+        ----------
+        csv_file : str
+            The CSV file containing the geometry details.
+            Expected column headers are dependent on the geometry type:
+            - if ``geo_type=='cylinder'``, expected fields are ``'xc'`` (base
+            center x coordinate),``'yc'`` (base center y coordinate), ``'zc'``
+            (base center z coordinate), ``'height'``, ``'radius'``,
+            ``'iradius'`` (inner radius for hollow cylinder), ``'plane'``
+            (plane on which the cylinder base lies).
+        header_line : int, optional
+            The line marking the start of data.
+            The default value is ``2``.
+        column_mapping : dict, optional
+            Map of the csv columns names to the accepted column names.
+            The default value is ``None``.
+        geo_type: str, optional
+            Type of geometry that the csv file describe. Accepted value is: ``"cylinder"``.
+            Default is ``"cylinder"``.
+        unit : str , optional
+            Unit of values used to create geometry. Default unit is ``"mm"``.
```

Comment:
```suggestion
        csv_file : str
            CSV file containing the geometry data.
            Expected column headers are dependent on the geometry type:
            - If ``geo_type=='cylinder'``, expected fields are ``'xc'`` (base
            center x coordinate),``'yc'`` (base center y coordinate), ``'zc'``
            (base center z coordinate), ``'height'``, ``'radius'``,
            ``'iradius'`` (inner radius for hollow cylinder), and ``'plane'``
            (plane on which the cylinder base lies).
        header_line : int, optional
            Line marking the start of data.
            The default is ``2``.
        column_mapping : dict, optional
            Map of the CSV column names to the accepted column names.
            The default is ``None``.
        geo_type: str, optional
            Type of geometry that the CSV file describe. The default is ``"cylinder"``.
        unit : str , optional
            Unit of values to use to create the geometry. The default is ``"mm"``.
```
We drop an initial article (The, A, An) for the brief description so that it begins with an important word. However, all sentences following the brief description use articles. For the first parameter, I wasn't sure why you called out ``If ``geo_type=='cylinder'`...." What happends if it is other "cylinder"? It seems ``cylinder is the only type mentioned.